first.css

```diff
@@ -1,3 +1,3 @@
-.header{
-    color: red;
-}
\ No newline at end of file
+.header {
+  color: red;
+}
```

Comment:
- [ ] A proper branch name should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). Please read [here ](https://codingsight.com/git-branching-naming-convention-best-practices/) about good branch naming conventions. 
To fix this, please: 
- create a new branch
- Update it with project code
- Push to GitHub Repo
- Open a new pull request. 
- In the new Pull request, add a link to the old (this) pull request to help the next reviewer track the changes requested.